test/mpi/maint/jenkins/xfail.conf

```diff
@@ -40,6 +40,7 @@ ch4 * * * * sed -i "s+\(^idup_comm_gen .*\)+\1 xfail=ticket0+g" test/mpi/threads
 * * * ch4:ucx * sed -i "s+\(^win_large_shm .*\)+\1 xfail=ticket0+g" test/mpi/rma/testlist
 * * * ch4:ucx * sed -i "s+\(^strided_putget_indexed_shared .*\)+\1 xfail=ticket0+g" test/mpi/rma/testlist
 * * * ch4:ucx * sed -i "s+\(^bcastlength .*\)+\1 xfail=ticket0+g" test/mpi/errors/coll/testlist
+* * * ch4:ucx * sed -i "s+\(^gather_big .*\)+\1 xfail=issue3770+g" test/mpi/coll/testlist
 # am-only failures
```

Comment:
We have now 5 xfail entries of `gather_big` and start to lose track on which is still on. Should we group them together?

Reply:
and we might as well add `issue3770` tag to all of them.

Reply:
The rest are not covered by that issue, so that'd be incorrect.

Reply:
Also, the rest are not `gather_big`.